github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRCommentEvent.java

```diff
@@ -44,15 +44,16 @@ public GitHubPRCommentEvent(String comment) {
     @Override
     public GitHubPRCause check(GitHubPRTrigger gitHubPRTrigger, GHPullRequest remotePR,
                                @CheckForNull GitHubPRPullRequest localPR, TaskListener listener) {
-        if (isNull(localPR) || isNull(localPR.getLastCommentCreatedAt())) {
```

Comment:
Yes. On second check this this will be skipped and https://github.com/KostyaSha/github-integration-plugin/pull/153/files/83bf7c0a460b1eed0a03db81704776474fedd5a6#diff-5cbd6dc7a8facee8006e0a0c89b42635L55 will be tested next and trigger the job if the second comment (or first, not shure) contains the key phrase. But we drop the first comment that may be a key phrase. I'm trying to move this `test expression` to the next statement to check that we are faced to first comment or there were other and we need to compare they timestamps (sorry for terminology).